ui/raidboss/data/07-dt/trial/queen-eternal.txt

```diff
@@ -0,0 +1,250 @@
+279.4 "Legitimate Force 2" Ability { id: ["8F22", "8F23"], source: "Queen Eternal" }
+293.5 "Prosecution Of War" Ability { id: "8EFA", source: "Queen Eternal" }
+308.6 "Legitimate Force" Ability { id: "8F20", source: "Queen Eternal" } window 20,20 forcejump "castellationHold"
+
+
+
+400.0 label "castellationBlock"
+400.0 "--sync--" StartsUsing { id: "8EFF", source: "Queen Eternal" } window 175,10
+404.7 "Virtual Shift" Ability { id: "8EFF", source: "Queen Eternal" }
+412.8 "Castellation 1" Ability { id: "8F05", source: "Queen Eternal" }
+420.5 "Besiegement 1" Ability { id: "8F08", source: "Queen Eternal" }
+430.2 "Castellation 2" Ability { id: "8F05", source: "Queen Eternal" }
+437.9 "Besiegement 2" Ability { id: "8F08", source: "Queen Eternal" }
+446.3 "Besiegement 3" Ability { id: "8F08", source: "Queen Eternal" }
+453.3 "Legitimate Force 1" Ability { id: ["8F1E", "8F1F", "8F20", "8F21"], source: "Queen Eternal" }
+456.4 "Legitimate Force 2" Ability { id: ["8F22", "8F23"], source: "Queen Eternal" }
+468.5 "Virtual Shift" Ability { id: "8F00", source: "Queen Eternal" } window 100,10
+476.7 "Coronation" Ability { id: "8F15", source: "Queen Eternal" }
+487.8 "Legitimate Force 1" Ability { id: ["8F1E", "8F1F", "8F20", "8F21"], source: "Queen Eternal" }
+490.9 "Legitimate Force 2" Ability { id: ["8F22", "8F23"], source: "Queen Eternal" }
+495.8 "Brutal Crown" Ability { id: "8F19", source: "Queen Eternal" }
+498.9 "--sync--" Ability { id: "8F16", source: "Queen Eternal" } # Lock and Key
+
+# It's possible there's a loop here,
+# but across More Than Five logs, we haven't seen one.
+# Royal Domain doesn't seem to be option, but Prosecution Of War definitely is.
+510.0 "Royal Domain" Ability { id: "8EFB", source: "Queen Eternal" }
+512.1 "Prosecution Of War?" Ability { id: "8EFA", source: "Queen Eternal" }
+
+# Absolute Authority madness
+# Seems to be at a 55-51% HP threshold?
+581.3 "--sync--" StartsUsing { id: "9A6B", source: "Queen Eternal" } window 500,10
+600.0 "Absolute Authority (pre-cast)" Ability { id: "9A6B", source: "Queen Eternal" }
+606.2 "Absolute Authority (flare)" Ability { id: "9A5E", source: "Queen Eternal" }
+610.3 "Absolute Authority (puddles 1)" #Ability { id: "8F0E", source: "Queen Eternal" }
+610.6 "Absolute Authority (abandonment)" Ability { id: "9A5F", source: "Queen Eternal" }
+614.5 "Authority\'s Hold" # LosesEffect { effectId: "1022" }
+614.5 "Absolute Authority (puddles 2)" #Ability { id: "8F0E", source: "Queen Eternal" }
+617.7 "Absolute Authority (gaze)" Ability { id: "8F10", source: "Queen Eternal" }
+618.9 "Absolute Authority (puddles 3)" #Ability { id: "8F0E", source: "Queen Eternal" }
+620.3 "Absolute Authority (raidwide)" Ability { id: "8F0D", source: "Queen Eternal" }
+639.5 "Divide and Conquer x8 (single)" Ability { id: "8F1C", source: "Queen Eternal" } duration 8
+651.1 "Divide and Conquer (all)" #Ability { id: "8F1D", source: "Queen Eternal" }
```

Comment:
These could use a `Spread` call.

Reply:
This is actually going to be slightly more complex than I had expected. There's no castbar for individual usages, and the headmarkers are the weird reversed source/target thing I've seen for laser mechanics elsewhere in Dawntrail. It looks like the player ID is in the `data0` field, so we can *get* it, but I just need to be sure I'm correctly comparing player data since I haven't used the `data.party` property all that much.

Reply:
Maybe just a generic `Spread` trigger with no condition/target for now then?